### ISSUE TITLE
Menu: Improve menu

### ISSUE THREAD

**Issue by alfetopito** [gnosis/cowswap#2456]
Users don't find it, and is hard to find info about CowSwap
Also, has too many elements. 
Many better to have it similar to the 1inch one

**Comment by W3stside** [comment]
By 1inch-like, do you mean:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/21335563/155968980-00245378-c557-4db0-a735-42ad135e5f44.png">

**Comment by alfetopito** [comment]
> By 1inch-like, do you mean: <img alt="image" width="303" src="https://user-images.githubusercontent.com/21335563/155968980-00245378-c557-4db0-a735-42ad135e5f44.png">

Yep. And not just `more`, also all the other sections like bridges and etc.
